### ISSUE TITLE
Fork Icon on previews in Viz Templates

### ISSUE THREAD

**Issue by nitanagdeote** [datavis-tech/vizhub-issue-tracker#379]
In viz templates a fork icon next to upvote icon will be of great help. I always forget to fork the template and then it shows the red alert.

**Comment by curran** [comment]
Agreed. It's definitely a usability problem at the moment, that when you click on a template within "Create viz", the fork flow is not clear/obvious.

**Comment by curran** [comment]
This is what Sam Vogt designed for this. I'd like to incrementally make the previews closer and closer to this design. One potential issue is that each of these previews takes up a lot more space than what we have now.

![image](https://user-images.githubusercontent.com/68416/86486516-773cbf00-bd29-11ea-8393-38eec1923b14.png)

**Comment by curran** [comment]
Older designs from Alec:

![image](https://user-images.githubusercontent.com/68416/86486586-a5ba9a00-bd29-11ea-89b3-bee774bdb79c.png)
